virtwho/register.py

```diff
@@ -443,6 +443,302 @@ def associate(self, host_name, guest_uuid):
+            in virtwho.ini as default.
+        """
+        register = get_register_handler('satellite')
+        self.org = org or register.default_org
+        self.activation_key = activation_key or register.activation_key
+        self.ssh = SSHConnect(host=register.server,
+                              user=register.ssh_username,
+                              pwd=register.ssh_password)
+        self.org_id = self.org_id()
+        self.api = f'https://{register.server}'
+        self.auth = (register.username, register.password)
+
+    def org_id(self, org=None):
+        """
+        Get the organization id by organization label.
+        :param org: organization label, use the org when instantiate
+        the class as default
+        :return: organization id
+        """
+        org = org or self.org
+        ret, output = self.ssh.runcmd(f'hammer organization info '
+                                      f'--label "{org}" '
+                                      f'--fields Id')
+        if ret == 0 and 'Id' in output:
+            org_id = output.split(':')[1].strip()
+            return org_id
+        raise FailException(f'Failed to get the organization id for {org}')
+
+    def org_create(self, name, label, description=None):
+        """
+        Create a new organization.
+        :param name: the name of the organization.
+        :param label: the label of the organization.
+        :param description: the description for the organization.
+        :return: True or raise Fail
+        """
+        description = description or ''
+        _, output = self.ssh.runcmd(f'hammer organization create '
+                                    f'--name "{name}" '
+                                    f'--label "{label}" '
+                                    f'--description "{description}"')
+        if 'Organization created' in output:
+            logger.info(f'Succeeded to create organization:{name}')
+            return True
+        if (
+                'Name has already been taken' in output
+                and
+                'Label has already been taken' in output
+        ):
+            logger.info(f'The organization:{name} already existed')
+            return True
+        raise FailException(f'Failed to create organization:{name}')
+
+    def org_delete(self, label):
+        """
+        Delete an organization by organization label.
+        :param label: organization label
+        :return: True or raise Fail
+        """
+        _, output = self.ssh.runcmd(f'hammer organization delete '
+                                    f'--label "{label}"')
+        if '100%' in output:
+            logger.info(f'Succeeded to delete organization:{label}')
+            return True
+        if 'organization not found' in output:
+            logger.info(f'The organization:{label} does not exist already')
+            return True
+        raise FailException(f'Failed to delete organization:{label}')
+
+    def host_id(self, host):
```

Comment:
I think the param```host``` should can be make clear before call the function, host_name or uuid or hwuuid, so there is no need define all ```host_name, host_uuid, host_hwuuid```